src/base/ddl_log.c

```diff
@@ -98,7 +98,7 @@ struct t_ddl_audit_handle
   bool jsp_mode;
 };
 
-static T_DDL_AUDIT_HANDLE ddl_audit_handle = { 0x00, };
+static T_DDL_AUDIT_HANDLE ddl_audit_handle;
```

Comment:
This has been modified because many warning messages were generated during the compilation process.